tests/type_inference/test_literals.py

```diff
@@ -47,6 +47,29 @@ def test_heterogeneous_lists(lst):
     assert [List[Any]] == result
 
 
+@given(PRIMITIVE_TYPES.flatmap(lambda s: hs.dictionaries(s(),s(),  min_size=1)))
+def test_homogeneous_dict(dict):
```

Comment:
Don't use the parameter name `dict`, as it shadows the Python type. You can use "dictionary" instead.